### PR TITLE
chore(#177043513) resend verification email

### PR DESCRIPTION
**Pull request by thepatrickniyo**
### **What doe this PR **do****
- It added resend Verification Email chore

### **How should this be tested**
-  clone the repository
-  run  `npm install` to install dependencies
-  switch to brache `ch-resendVerificationEmail-#177043513`
- run `npm run start` to start the server
-  signup a new user
-  call api `localhost:5000/user/resendVerificationEmail` [POST]  to test the chore

### **Any background information**
- The request body should take only email

### **Expected output**
- a 200 status code is expected if the resending verification is successful

### **Relevant Trello story**
- #177043513

### **Any screenshot**
image

### CODE REVIEW

File: src/controllers/user.js [hunk 1]
```diff
@@ -145,3 +145,23 @@ export const verifyUser = async (req, res) => {
...
+    return res.status(401).json({message: res.__("Unauthorized")})
+  }
+   if(isSignedup.dataValues.isVerified == true){
+     return res.status(400).json({message: res.__("Email already verified")});
+   }
+   const token = jwt.sign(JSON.parse(JSON.stringify(isSignedup.dataValues)), process.env.JWT_KEY, { expiresIn: '1h' });
+          jwt.verify(token, process.env.JWT_KEY, () => {});
+  await sendVerificationEmail(isSignedup.dataValues.firstname,email,token);
+  return res.status(200).json({message: res.__("Confirmation email resent to "+email)})
+}
```
Comment (SimonImanigirimpuhwe) [inline src/controllers/user.js]: Good implementation

**Review by SimonImanigirimpuhwe — APPROVED** [review]

**Review by leny62 — APPROVED** [review]
Kindly, check on code climate, there is some testing vulnerabilities. Otherwise, good work!

**Review by leny62 — APPROVED** [review]
Kindly, check on code climate, there is some testing vulnerabilities. Otherwise, good work!

**Review by furebo — APPROVED** [review]
Good , now the travis is also passing

**Pull request merged**